spec.bs

```diff
@@ -569,3 +569,50 @@ Alternatively, we can add the snippet at the beginning of the following HTML spe
 </div>
 
 <p class="note">[=Validate a module script URL=] is applied to all module URLs before they start loading, even in paths where [=resolve a module specifier=] and import maps are not applied (e.g. `<script src="..." type="module">`).</p>
+
+<hr>
+<h2 id="internet-media-type-registration">Appendix A: Internet Media Type Registration</h2>
+
+This appendix provides the provisional registration of the Internet Media Type `application/importmap+json` in accordance with [[RFC6838]].
+
+: Type name:
+:: application
+: Subtype name:
+:: importmap+json
+: Required parameters:
+:: N/A
+: Optional parameters:
+:: N/A
+: Encoding considerations:
+:: 8bit
+: Security considerations:
+:: Since Import Maps have the ability to direct which module resolutions are to be provided to all ECMAScript module imports of a given ECMAScript environment, control of the Import Map should be considered to be execution-level application access. Integration with existing policy systems, including for example CORS and CSP, can be used to mitigate and restrict unwanted target URLs from being executed. In addition, the specification states that only those Import Maps served to browsers with the `application/importmap+json` MIME type will be executed.
+: Interoperability considerations:
+:: Backwards compatibility will be a necessity for any new features added to the format, and some handling for this has been incorporated into the design of the specification.
+: Published specification:
+:: https://wicg.github.io/import-maps/
+: Applications that use this media type:
+:: This is a browser-specific MIME type but may also apply to other ECMAScript environments.
+: Fragment identifier considerations:
+:: N/A
+: Restrictions on usage:
+:: N/A
+: Additional information:
+: Deprecated alias names for this type:
+:: N/A
+: Magic number(s)
+:: N/A
+: File extension(s):
+:: .importmap
+: Macintosh file type code:
+:: N/A
+: Object Identifiers:
+:: N/A
+: Person & email address to contact for further information:
+:: Guy Bedford (guybedford@gmail.com).
+: Intended usage:
+:: COMMON
+: Restrictions on usage:
+:: N/A
+: Change controller:
+:: W3C
```

Comment:
I'm not sure how much sense this makes given that import maps will become part of the WHATWG HTML specification.

RFC 6838 doesn't say much about this field. Perhaps just your name would work?